### DISCUSSION TITLE
Verification of Free Energy Results

### DISCUSSION THREAD

**Discussion by alanjchang** [Valdes-Tresanco-MS/gmx_MMPBSA#466] [General]
How do I know if the free energy values I am getting from gmx_MMPBSA on a protein-aptamer complex is reasonable?

**Comment by marioernestovaldes** [comment]
You could use

-Data from the literature for similar complexes (experimental and in silico)
-Comparison with some standard with known interaction

**Reply by alanjchang** [reply]
However, free energy predictions do not tend to predict absolute binding free energies well, so even if I had kD values from experiments (which I do), how can I tell that my free energy values are reasonable? Most of my kD values for the protein-aptamer complexes that I want to run MMPBSA on are in the nanomolar (nM) range, which according to this https://www.novoprolabs.com/tools/deltag2kd, corresponds to a free energy value of -12 to -8.

Comparison with some standard known interaction might be tricky too because how would I know from complex to complex that my process is correct? I.e. even if I had some interaction that I knew MMPBSA accurately described the free energy of, how do I know that there were not any errors specific to the complex that I am testing at hand?

**Comment by marioernestovaldes** [comment]
End-point free energy methods like MMPBSA or MMGSBA are usually more suitable to calculate relative binding free energies, that is binding free energy differences between two or more complexes. The method perhaps is not able to reproduce the absolute energies but could reproduce the difference between two energy values. There is a lot of literature explaining the pros and cons of end-point free energy methods. Here is a good starting point:

https://valdes-tresanco-ms.github.io/gmx_MMPBSA/dev/introduction/#literature

**Reply by alanjchang** [reply]
Are there any other indicators besides some measure of binding affinity that tell me whether or not I have my GROMACS simulation or my gmx_MMPBSA process correctly. For example, can the .xtc trajectory file output from my GROMACS simulation tell me if my protein-aptamer complex was preprocessed correctly for gmx_MMPBSA? What the geometry of the output .gro file or the .tpr file? I guess I am just looking for ways to validate my procedure and results.

**Comment by marioernestovaldes** [comment]
The usual go-to in these cases is, as I mentioned earlier, contrasting the results in silico with experimental measurements. For example, compare calculated in silico relative binding free energies with experimental ones. This is universally accepted in the literature as a way to "validate" your findings.

**Reply by alanjchang** [reply]
What if I am using GROMACS/gmx_MMPBSA to test if these simulation accurately predict experimental relative binding affinities? If I am getting results from my simulations that are inconsistent with experimental relative binding affinities, how do I know if it's due to error/inaccuracies intrinsic to GROMACS/gmx_MMPBSA or if it's due to me running the GROMACS/gmx_MMPBSA procedure incorrectly?

**Comment by marioernestovaldes** [comment]
By learning how to use GROMACS/gmx_MMPBSA first, you would be in a better position to judge the results. It is not recommended to use these programs as a black box.
